src/PlayerHitBillboardPhysicsComponent.cpp

```diff
@@ -0,0 +1,19 @@
+#include "PlayerHitBillboardPhysicsComponent.h"
+
+#include "GameManager.h"
+#include "GameObject.h"
+#include "GameWorld.h"
+
+void PlayerHitBillboardPhysicsComponent::updatePhysics(float deltaTime) {
+	currentLifeTime += deltaTime;
+
+	if (currentLifeTime > maxLifeTime) {
+		GameManager& gameManager = GameManager::instance();
+		GameWorld& world = gameManager.getGameWorld();
+
+		world.rmDynamicGameObject(holder_);
+	}
+	else {
+		holder_->setScale(glm::vec3(1.0) + glm::vec3(sin(currentLifeTime * M_PI)));
```

Comment:
Got home and tried to build and it did err for GCC. Just add a temp `glm::vec3` var above this line, set it, then pass it to `holder_->setScale` and that should fix it.